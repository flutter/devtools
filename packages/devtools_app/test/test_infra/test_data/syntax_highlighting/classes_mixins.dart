// Copyright 2022 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

class A {
  String get name => '';
  set name(String value) {}

  void method() {}
  void get() {}
  void set() {}
}

mixin MyMixin<T> on List<T> {}
