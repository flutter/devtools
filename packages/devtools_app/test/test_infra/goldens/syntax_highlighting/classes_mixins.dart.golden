>// Copyright 2022 The Chromium Authors. All rights reserved.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// Use of this source code is governed by a BSD-style license that can be
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>// found in the LICENSE file.
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ comment.line.double-slash.dart
>
>class A {
#^^^^^ keyword.declaration.dart
#      ^ support.class.dart
>  String get name => '';
#  ^^^^^^ support.class.dart
#         ^^^ keyword.declaration.dart
#             ^^^^ entity.name.function.dart
#                     ^^ string.interpolated.single.dart
#                       ^ punctuation.terminator.dart
>  set name(String value) {}
#  ^^^ keyword.declaration.dart
#      ^^^^ entity.name.function.dart
#           ^^^^^^ support.class.dart
>
>  void method() {}
#  ^^^^ storage.type.primitive.dart
#       ^^^^^^ entity.name.function.dart
>  void get() {}
#  ^^^^ storage.type.primitive.dart
#       ^^^ entity.name.function.dart
>  void set() {}
#  ^^^^ storage.type.primitive.dart
#       ^^^ entity.name.function.dart
>}
>
>mixin MyMixin<T> on List<T> {}
#^^^^^ keyword.declaration.dart
#      ^^^^^^^ support.class.dart
#             ^ other.source.dart
#              ^ support.class.dart
#               ^ other.source.dart
#                 ^^ keyword.control.catch-exception.dart
#                    ^^^^ support.class.dart
#                        ^ other.source.dart
#                         ^ support.class.dart
#                          ^ other.source.dart
